src/Microsoft.IdentityModel.Tokens/CryptoProviderFactory.cs

```diff
@@ -45,6 +46,9 @@ namespace Microsoft.IdentityModel.Tokens
     /// </summary>
     public class CryptoProviderFactory
     {
+        /// <summary>
+        /// Returns a new <see cref="CryptoProviderFactory"/> instance.
```

Comment:
This is a static, so it's not newing up an instance. This should read "Returns a default CryptoProviderFactory instance."